frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -64,6 +92,23 @@ export default {
       type: Array,
       default: () => [],
     },
+    header: {
+      type: String,
+      required: false,
+      default: undefined
+    },
+    showInfoMessage: {
```

Comment:
Is this prop required? It's only used in a v-else block if there's no header so it's redundant based on how it's used.

Reply:
It is, for the UI layout on applications we now don't want to show the doc validation info message on each instance of the upload component, but rather show once outside of the component. This prop allows us to control this behavior.

Reply:
Removed to be implied from documentType